src/main/java/com/icthh/xm/ms/entity/domain/XmEntity.java

```diff
@@ -130,10 +130,10 @@
 
     /**
      * This is i18n name of Entity.
-     * TODO: change data type
      */
-    @ApiModelProperty(value = "This is i18n name of Entity. TODO: change data type", required = true)
+    @ApiModelProperty(value = "This is i18n name of Entity.", required = true)
     @Column(name = "name", nullable = false)
+    @Field(type = FieldType.Keyword)
```

Comment:
It will lead to errors in existing projects. If you need to use 'name' field for for sorting or aggregations you can use 'name.keyword'. Also you can change the field type in the mapping.json file.

Reply:
ok, thanks..